pkg/account/downloader/internal/http/client.go

```diff
@@ -41,6 +41,29 @@ func (c *Client) GetUsers(ctx context.Context, accountUUID string) ([]accountman
 	return r.Items, nil
 }
 
+func (c *Client) GetServiceUsers(ctx context.Context, accountUUID string) ([]accountmanagement.ExternalServiceUserDto, error) {
+	serviceUsers := []accountmanagement.ExternalServiceUserDto{}
+	const pageSize = 10
```

Comment:
There are max 10 pages?

Reply:
We need to pick something here, yes ;)

Reply:
But this is actually the page size, that is, the number of service users retrieved per request. There is no maximum number of pages retrieved.